docs/developer-guide.md

```diff
@@ -22,7 +22,7 @@ You can build all images from source.
 
 ## Implement new suggestion algorithm
```

Comment:
I think, we should also change path here: https://github.com/kubeflow/katib/pull/452/files#diff-ba23e0f18c79cb873ac5375367524cfaL20.

Reply:
done